### PR TITLE
Feature/delete/venue review

### PR DESCRIPTION
**Pull request by abyan-n**
# Description

[Link to Ticket](insert the link to your ticket inside the parenthesis here)

Please include a summary of the changes and the related issue. Please also
include relevant motivation, context, and images!
https://trello.com/c/H37iULmB/19-delete-api-venues-reviews-reviewid

Added a delete endpoint to delete a review for a venue.

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. If they are unit
tests, provide the file name the tests are in. If they are not unit tests,
describe how you tested the change.

Postman, checked with valid ID, invalid ID, invalid path params.

# Checklist

- [X ] I have performed a self-review of my code
- [ ] I have reached out to another developer to review my code
- [ X] I have commented my code, particularly in hard-to-understand areas
- [ X] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: backend/internal/handlers/venues/routes.go [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package venues
+
+//NOTE: This is an example usage for auth demonstration purposes. In real configurations (beyond login) all route groups should be protected
```
Comment (CamPlume1) [inline backend/internal/handlers/venues/routes.go:3]: Probably delete legacy comments

File: backend/internal/storage/storage.go [hunk 2]
```diff
@@ -27,6 +28,11 @@ type UserRating interface {
 	GetAllUserRatings(context.Context, uuid.UUID) ([]models.UserRating, error)
 }
 
+type Venues interface {
```
Comment (CamPlume1) [inline backend/internal/storage/storage.go:31]: I think we should combine Venue and Venue Ratings interfaces. IMO they provide functionality over the same data structure and belong together

**Review by CamPlume1 — CHANGES_REQUESTED** [review]
Small notes:

1. Combine Interface is the blocking change here. Deleting legacy comment would be a nice bonus 👍

File: backend/internal/storage/storage.go [hunk 3]
```diff
@@ -30,6 +30,8 @@ type UserRating interface {
 	GetAllUserRatings(context.Context, uuid.UUID) ([]models.UserRating, error)
 }
 
-type VenueRatings interface {
+type Venues interface {
```
Comment (wyattchris) [inline backend/internal/storage/storage.go:33]: looks great!!

**Review by wyattchris — COMMENTED** [review]
interface change resolved

**Pull request merged**